src/common/metrics/vaccinations.tsx

```diff
@@ -68,13 +68,25 @@ function renderStatus(projections: Projections): React.ReactElement {
   const peopleVaccinated = formatInteger(info.peopleVaccinated);
   const percentVaccinated = formatPercent(info.ratioVaccinated, 1);
 
+  const cappedVaccinatedCopy =
+    info.ratioInitiated >= 0.95 ? (
+      <Fragment>
+        <br />
```

Comment:
There must be a better way of adding a line break, but this is all I could find that worked

Reply:
that worked great! It did add the new line at the end if we're okay with that

<img width="940" alt="image" src="https://user-images.githubusercontent.com/55333380/148857670-a4b52bb6-e361-4c74-b5e3-7a21843b4fcf.png">